dist/package.json

```diff
@@ -16,7 +16,7 @@
     "StyledFirebaseAuth.js",
     "StyledFirebaseAuth.js.map"
   ],
-  "main": "index.js",
+  "main": "dist/index.js",
```

Comment:
Should just be `index.js` here no?

Reply:
Fix: https://github.com/firebase/firebaseui-web-react/blob/e9cf37bf5a8b621d92399ed4ca39185ecec65a50/dist/package.json#L19